src/Commands/ProjectMergeRequest.php

```diff
@@ -109,7 +123,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         );
 
         $response = $io->choice(
-            'Please select merge request to checkout',
+            'Select merge request to checkout',
             $choices,
         );
         $mrId = array_search($response, $choices, true);
```

Comment:
Can `$mrId` be null here if we select illegal choice?

Reply:
choice question from symfony console wont continue until a valid `$choice` is entered.